Documentation/EyeTracking/EyeTracking_TargetSelection.md

```diff
@@ -69,9 +69,13 @@ Here is a simple example from [ColorTap.cs](xref:Microsoft.MixedReality.Toolkit.
 
 ### Selecting a focused hologram 
 
-To select focused holograms, use PointerHandler to listener for input event to confirm a selection. For example, adding the _IMixedRealityPointerHandler_ will make them react to simple pointer input. The _IMixedRealityPointerHandler_ interface requires implementing the following three interface members: _OnPointerUp_, _OnPointerDown_, and _OnPointerClicked_.
+To select a focused hologram, use _PointerHandler_ to listen for input events to confirm a selection. 
+For example, adding the _IMixedRealityPointerHandler_ will make them react to simple pointer input. 
+The _IMixedRealityPointerHandler_ interface requires implementing the following three interface members: 
+_OnPointerUp_, _OnPointerDown_, and _OnPointerClicked_.
```

Comment:
Love the character count enforcement thank you!

This makes looking at diffs WAY easier in the future.